grails-app/i18n/messages_es.properties

```diff
@@ -482,7 +482,7 @@ default.notes.label=Notes
 default.null.message=La propiedad [{0}] de la clase [{1}] no puede ser nulo
 #default.number.format                 = 0
 default.numItems.label=\# items
-default.number.format=\# \# \#, \# \# \#, \# \# \#0.00
+default.number.format=\# \# \#, \# \# \#, \# \# \#
```

Comment:
@jmiranda this is the second time crowdin is trying to remove this part, which was added on purpose.

@drodzewicz maybe we can modify it through crowdin?

Reply:
yes, I think this is our only option at the moment

Reply:
Even after updating this label directly in crowdin, which I did yesterday, I still don't see an updated commit in this PR